crates/threshold-signature-server/src/attestation/api.rs

```diff
@@ -77,6 +78,30 @@ pub async fn attest(
     Ok(StatusCode::OK)
 }
 
+/// Retrieve a quote by requesting a nonce from the chain and return the quote in the HTTP response
+/// body.
+/// This is used by node operators to get a quote for use in the `validate`, `change_endpoint`
+/// and `change_tss_accounts` extrinsics.
+pub async fn get_attest(
```

Comment:
```suggestion
pub async fn request_attestation(
```

To match the extrinsic?